cmd/flags.go

```diff
@@ -156,6 +156,10 @@ func FlagProcessType(processType *string) {
 	flagSet.StringVar(processType, "process-type", os.Getenv(EnvProcessType), "default process type")
 }
 
+func FlagNoColor(skip *bool) {
+	flagSet.BoolVar(skip, "no-color", false, "disable color output")
```

Comment:
We should add an environment variable `CNB_NO_COLOR` and use it to set the default here. This is especially for things that might not be specified in the platform API, as it allows platforms to optimistically set it w/o worrying that it will cause a failure if it is unsupported by a given lifecycle.